app/views/content/train-to-be-a-teacher/classroom-placements.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Teacher training school placements"
+heading: "Teacher training school placements"
+subcategory: Postgraduate teacher training
+description: |-
+  Learn about teacher training school placements, including how long they are, what trainees will do on placements, and the support available. 
+navigation: 20.20
+navigation_title: School placements
+navigation_description: Learn about teacher training school placements, including how long they are, what trainees will do on placements, and the support available. 
+quote:
+  q-giorgio:
+    text: "The highlight of my placements was getting to teach!"
+    name: "Giorgio, biology teacher"
+    classes: quote--indent
+promo_content:
+    - content/train-to-be-a-teacher/promos/mailing-list-promo-itt
+    
+---
+
+School placements are a crucial part of your postgraduate teacher training. Placements are a chance to:
+
+* get experience in the classroom
+* immerse yourself in your placement schools
+* get to know your pupils and colleagues
+* start your career as a teacher
+
+## How long are placements?
+
+School placements last for a minimum of 24 weeks in at least 2 schools. This is to give you a varied and thorough experience in the classroom. 
```

Comment:
I find this slightly confusing - is it 24 weeks in each of 2 schools or 24 weeks in total spread across 2 schools? (You will be able to word this better than I can!)